eng/testing/AppleRunnerTemplate.sh

```diff
@@ -38,15 +35,25 @@ while true; do
         sleep 5
     fi
 done
+# Restart the simulator to make sure it is tied to the right user session
+xcode_version=11.4
+xcode_path="/Applications/Xcode${xcode_version/./}.app"
+simulator_app="$xcode_path/Contents/Developer/Applications/Simulator.app"
+pid=`ps aux | grep "$simulator_app" | grep -v grep | tr -s ' ' | cut -d ' ' -f 2`
+if [ ! -z "$pid" ]; then
+    sudo kill "$pid"
+fi
+open -a "$simulator_app"
+export XHARNESS_OUT="$EXECUTION_DIR/xharness-output"
 
-XHARNESS_OUT="$EXECUTION_DIR/xharness-output"
-
-dotnet xharness ios test --app="$APP_BUNDLE" \
-    --targets=$TARGET \
+dotnet xharness ios test \
```

Comment:
Let's match this what we did for `Browser` where the `RunCommands` are defined in `tests.mobile.targets` that way we can preserve xunit arguments like `-notrait Category=IgnoreForCI` or others like `Outerloop`.